### PR TITLE
e2e refactor:added TestDeeplinkChatProfile medium group

### PR DESCRIPTION
**Pull request by pavloburykh**
added TestDeeplinkChatProfile medium group

List of tests that have been refactored/put in the group: 

@marks.testrail_id(5781)
TODO: merge with other 1-driver medium e2e
def test_deep_link_with_invalid_user_public_key_own_profile_key(self)

@marks.testrail_id(5441)
TODO: merge with other 1-driver medium e2e
def test_deep_link_open_user_profile(self)

@marks.testrail_id(5442)
TODO: merge with other 1-driver medium e2e
def test_deep_link_open_dapp(self)

@marks.testrail_id(5498)
TODO: split into separate parts and include to group
def test_share_user_profile_url_public_chat(self)

@marks.testrail_id(6320)
TODO: to unite it with 5757
def test_chat_can_start_from_suggestions_using_search(self)

@marks.testrail_id(5757)
TODO: to unite it with 6320 and cut the part about offline - to add it to the end of group for instance
def test_chat_search_on_home(self)

@marks.testrail_id(5322)
TODO: merge with other 1-driver medium e2e
def test_chat_input_delete_cut_and_paste_messages(self)

@marks.testrail_id(5426)
TODO: add to any group with history in public chat
def test_chat_public_clear_history_via_options_and_long_press(self)

### CODE REVIEW

File: test/appium/tests/medium/test_deeplink_chat_share_profile.py [hunk 1]
```diff
@@ -0,0 +1,347 @@
...
+                browser.element_by_text_part(expeceted_text_2).is_element_present()):
+            self.errors.append("Can't share link to URL")
+
+        browser.click_system_back_button(1)
+
+        self.errors.verify_no_errors()
+
+    @marks.testrail_id(702248)
+    def test_chat_can_start_and_find_from_suggestions_using_search(self):
+        # chat = ChatView(self.drivers[0])
```
Comment (churik) [inline test/appium/tests/medium/test_deeplink_chat_share_profile.py]: remove

File: test/appium/tests/medium/test_deeplink_chat_share_profile.py [hunk 2]
```diff
@@ -0,0 +1,347 @@
...
+        for chat_name in options_chat_name, public_long_press:
+            chat = self.home.get_chat(chat_name).click()
+            if chat.chat_element_by_text(message).is_element_displayed():
+                self.errors.append('Messages in %s chat are shown after clearing history and relaunch' % chat_name)
+            chat.home_button.click()
+
+        self.errors.verify_no_errors()
+
+    @marks.testrail_id(702254)
+    # Should be the last in group as it disables network connection
```
Comment (churik) [inline test/appium/tests/medium/test_deeplink_chat_share_profile.py:326]: can be removed; but also can stay, no harm

**Review by churik — APPROVED** [review]
Awesome work, @pavloburykh !

File: test/appium/tests/medium/test_deeplink_chat_share_profile.py (hunk 1 above)
Comment (pavloburykh) [reply test/appium/tests/medium/test_deeplink_chat_share_profile.py]: removed

File: test/appium/tests/medium/test_deeplink_chat_share_profile.py (hunk 2 above)
Comment (pavloburykh) [reply test/appium/tests/medium/test_deeplink_chat_share_profile.py:326]: left this comment for a case if will work on this group and add some extra cases

**Comment by pavloburykh** [comment]
@churik thanx a lot for review. Merged.

**Pull request merged**